dashboard/test/lib/policies/lti_test.rb

```diff
@@ -88,7 +88,7 @@ class Policies::LtiTest < ActiveSupport::TestCase
     end
   end
 
-  test `force_iframe_launch? should return true for Schoology and false for other LMS platforms` do
+  test 'force_iframe_launch? should return true for Schoology and false for other LMS platforms' do
```

Comment:
Yikes! That tried to run `force_iframe_launch?` as a command and nothing caught this happening. Not sure anything could, but that's a little unnerving.

Reply:
Yep. The backticks would cause it to shell out that command whenever it is building the tests before they run.

Reply:
Lol OK! I see that now.